game.py

```diff
@@ -1,41 +1,487 @@
-# Module imports
-import pygame
-
-# File imports
-import boardSettings
-import shipPlacement
-import splashScreen
-import aiAgent
+import random
 
-from globalVariables import *
-from gameLogic import *
-from boardSettings import *
-from shipPlacement import *
+import pygame
 
 # Initialize Pygame
 pygame.init()
 
+# Set the dimensions of the window
+WINDOW_WIDTH = 800
+WINDOW_HEIGHT = 400
+WINDOW_SIZE = (WINDOW_WIDTH, WINDOW_HEIGHT)
+
+# Set the colors
+BLACK = (76, 63, 84)
+WHITE = (255, 255, 255)
+GRAY = (218, 165, 32)  # Mustard
+BLUE = (70, 130, 180)
+RED = (138, 51, 36)
+FIG = (138, 51, 36)  # for the ships
+GREEN = (175, 224, 144)
+
+# Set the size of the grids
+GRID_SIZE = 10
+ATTACK_SIZE = 20
+CELL_SIZE = WINDOW_HEIGHT // GRID_SIZE
+
+# Set the gap between the grids
+GAP = 10
+
+# Set the number of battleships
+SHIP_LENGTHS = [5, 4, 3, 3, 2]
+
+# Create the window
+window = pygame.display.set_mode(WINDOW_SIZE)
+pygame.display.set_caption("Battleship Game")
+
+# Create player 1 and player 2 grids
+player1_grid = [[0] * GRID_SIZE for _ in range(GRID_SIZE)]
+AI_grid = [[0] * GRID_SIZE for _ in range(GRID_SIZE)]
+
+# Array for AI's used attacks
+AI_shots = []
+AI_hits = []
+
+
+# Hit flag
+hitflag = 0
+
```

Comment:
Why did the separated functions came back? Dapat nakahiwalay na sya

Reply:
nagcommit na meee nung maayos naaa. maya ko na ulit tuloy HAHAHAHA